### PR TITLE
Update to Rails 4.0.13

### PR DESCRIPTION
**Pull request by eitoball**
It is on [staging server](http://api-staging.safecast.org) . If you find some issues, please add them.

## Issues

- [x] Pager link style
- [x] No description of check box on sign-in page
- [x] Form styling
  - [x] vertical to horizontal
  - [x] tooltip placement https://github.com/Safecast/safecastapi/pull/291#issuecomment-226686824

### CODE REVIEW

File: app/controllers/measurements_controller.rb [hunk 1]
```diff
@@ -98,5 +90,13 @@ def count
...
     respond_with @count
   end
-  
+
+private
+
+  def measurement_params
+    params.fetch(:measurement, {})
+      .permit(*%i(value unit location location_name device_id height
+                  surface :radiation latitude longitude captured_at
```
Comment (eitoball) [inline app/controllers/measurements_controller.rb]: typo https://safecast.slack.com/archives/api/p1466139436000334

**Comment by matschaffer** [comment]
Also tooltip placement (might get fixed with the "form styling") check:
bgeigie_import__10757____bgeigie_imports____safecast_api

**Comment by matschaffer** [comment]
This is looking pretty solid to me. @eitoball, @robouden, @jsidd - have a look around http://api-staging.safecast.org/ and let us know if you see anything amiss.

Thinking I'll roll this to prod tomorrow afternoon unless anything comes up.

**Comment by robouden** [comment]
Seems fine to me. Except that api-staging still times out on admin measurements. But that was expected. Shall I merge it and deploy on production?

**Comment by matschaffer** [comment]
Yeah, timeouts are expected. The staging DB is a bit too underpowered for that table.

**Comment by nokton** [comment]
Production also times out (at least for me) Better handle that as a separate issue and move on.

**Pull request merged**